src/main/java/net/minecraftforge/eventbus/api/Event.java

```diff
@@ -60,9 +60,9 @@ public Event() { }
      * Note:
      * Events with the Cancelable annotation will have this method automatically added to return true.
      */
-    public boolean isCancelable()
+    public final boolean isCancelable()
```

Comment:
Should update the docs as since the method is now `final` the annotation is the only way.